eclipse/sw360
# feat(UI): Added new site role 'CLEARING_EXPERT'

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
 * _Issue #820_ Added a new Role **Clearing Expert**, 
 * fixed bug where in adding **SITE role** to a user was not reflecting in couch db.
 * No changes in dependencies.

Issue: 

### Suggest Reviewer
> 
### How To Test?
* Adding a new `site role` to a user should reflect in couch db.
* All user with role `Clearing Expert`, should be able to view and edit the `Clearing Request` under their `department`.
* No additional tests implemented.

### Checklist
Must:

Signed-off-by: akapti <abdul.mannankapti@siemens.com>
